src/utils/payloadStringify.js

```diff
@@ -15,22 +17,33 @@ const TRUNCATED_TEXT = '...[too long]';
 
 const isNativeType = (obj) => nativeTypes.includes(typeof obj);
 
-export const keyToOmitRegexes = () => {
-  let regexesList = OMITTING_KEYS_REGEXES;
-  if (process.env[LUMIGO_SECRET_MASKING_REGEX_BACKWARD_COMP]) {
-    const parseResponse = parseJsonFromEnvVar(LUMIGO_SECRET_MASKING_REGEX_BACKWARD_COMP, true);
+const keyToRegexes = (
+  regexesList = OMITTING_KEYS_REGEXES,
+  backwardCompRegex = LUMIGO_SECRET_MASKING_REGEX_BACKWARD_COMP,
+  envDefinedRegexes = LUMIGO_SECRET_MASKING_REGEX
+) => {
+  if (process.env[backwardCompRegex]) {
+    const parseResponse = parseJsonFromEnvVar(backwardCompRegex, true);
     if (parseResponse) {
       regexesList = parseResponse;
     }
-  } else if (process.env[LUMIGO_SECRET_MASKING_REGEX]) {
-    const parseResponse = parseJsonFromEnvVar(LUMIGO_SECRET_MASKING_REGEX, true);
+  } else if (process.env[envDefinedRegexes]) {
+    const parseResponse = parseJsonFromEnvVar(envDefinedRegexes, true);
     if (parseResponse) {
       regexesList = parseResponse;
     }
   }
   return regexesList.map((x) => new RegExp(x, 'i'));
 };
 
+export const keyToOmitRegexes = () => {
+  return keyToRegexes();
+};
+
+export const WhitelistKeysRegexes = () => {
```

Comment:
```suggestion
export const whitelistKeysRegexes = () => {
```
Don't we have a linter that is suppose to catch this?